pages/dashboard/PKPDashboardHandler.php

```diff
@@ -158,6 +158,7 @@ public function index($args, $request)
 
         $versionStageOptions = [];
         $allVersionStages = VersionStage::cases();
+        usort($allVersionStages, fn($a, $b) => $a->order() <=> $b->order());
```

Comment:
This change improves the ordering of the dropdown options for Publication Stage when creating a new publication version, to follow the expected order of publication stages:
OJS: AO, SM, AM, PR, VoR
OMP: AO, SM, AM. PR, VoR
OPS: AO, SM